modules/nf-core/canu/main.nf

```diff
@@ -0,0 +1,48 @@
+process CANU {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::canu=2.2"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/canu:2.2--ha47f30e_0':
+        'quay.io/biocontainers/canu:2.2--ha47f30e_0' }"
+
+    input:
+    tuple val(meta), path(reads)
+    val mode
+    val (genomesize)
+
+    output:
+    tuple val(meta), path("*.report")			, emit: report
+    tuple val(meta), path("*.contigs.fasta")		, emit: assembly		, optional: true
+    tuple val(meta), path("*.unassembled.fasta")	, emit: contigs
```

Comment:
> Where applicable, the usage and generation of compressed files SHOULD be enforced as input and output

https://nf-co.re/developers/modules#general
would that be possible?

Reply:
I think it should be possible, let's try.

Reply:
Done